service/src/main/java/org/jboss/sbomer/service/rest/api/v1beta1/RequestsV1Beta1.java

```diff
@@ -0,0 +1,351 @@
+                                    value = "{\"type\": \"pnc-operation\", \"operationId\": \"A5WL3DFZ3AIAA\"}"),
+                            @ExampleObject(
+                                    name = "Errata Tool advisory",
+                                    description = "Requests manifest generation for the 12345 advisory",
+                                    value = "{\"type\": \"advisory\", \"advisoryId\": 12345}"),
+                            @ExampleObject(
+                                    name = "Container image",
+                                    description = "Requests manifest generation for a container image with iodentifier: registry.access.redhat.com/ubi9/ubi-micro:9.4",
+                                    value = "{\"type\": \"syft-image\", \"image\": \"registry.access.redhat.com/ubi9/ubi-micro:9.4\"}") //
+                    }))
+    @APIResponse(
+            responseCode = "202",
+            description = "Manifest generation successfully requested",
+            content = @Content(schema = @Schema(implementation = V1Beta1SbomGenerationRequestRecord.class)))
+    @APIResponse(
+            responseCode = "500",
+            description = "Internal server error",
+            content = @Content(mediaType = MediaType.APPLICATION_JSON))
+    public Response createFromConfig(Config config) throws Exception {
+        if (config == null) {
+            throw new ClientException("No config provided");
+        }
+
+        log.info("Validating config: {}", ObjectMapperProvider.json().writeValueAsString(config));
+
+        ValidationResult validationResult = configSchemaValidator.validate(config);
+
+        if (!validationResult.isValid()) {
+            throw new ClientException("Invalid config", validationResult.getErrors());
+        }
+
+        log.info("Provided config is valid!");
+
+        List<SbomGenerationRequest> requests = new ArrayList<>();
+
+        if (config instanceof AdvisoryConfig advisoryConfig) {
+            log.info("New Errata advisory request received");
+            requests.addAll(advisoryService.generateFromAdvisory(advisoryConfig.getAdvisoryId()));
+        } else if (config instanceof PncBuildConfig pncBuildConfig) {
+            log.info("New PNC build request received");
+            requests.add(sbomService.generateFromBuild(pncBuildConfig.getBuildId(), pncBuildConfig));
+        } else if (config instanceof DeliverableAnalysisConfig analysisConfig) {
+            log.info("New PNC analysis request received");
+            requests.add(sbomService.generateNewOperation(analysisConfig));
+        } else if (config instanceof SyftImageConfig syftImageConfig) {
+            log.info("New container image request received");
+            requests.add(sbomService.generateSyftImage(syftImageConfig.getName(), syftImageConfig));
+        } else if (config instanceof OperationConfig) {
+            log.info("New PNC operation request received");
+
```

Comment:
@goldmann I missed this, shouldn't it be `requests.add(sbomService.generateFromOperation(operationConfig));`  instead of not implemented?

Reply:
This should be now covered!